libs/ts/config-types/tests/config-decoding.spec.ts

```diff
@@ -1,173 +1,34 @@
-  ).name;
-  const chainlinkCompatibilityV2 = path.parse(
-    configFiles.chainlink_compatibility_v2,
-  ).name;
-  const evmContractsDeploymentV1 = path.parse(
-    configFiles.evm_contracts_deployment_v1,
-  ).name;
-
-  test('should decode v1 feeds config successfully', async () => {
-    {
-      expect(
-        async () => await decodeJSON({ name: feedConfigV1 }, FeedsConfigSchema),
-      ).not.toThrow();
-    }
-
-    {
-      const feedConfigV1Content = JSON.parse(
-        JSON.stringify(await readJSON({ name: feedConfigV1 })),
-      );
-      expect(() => decodeFeedsConfig(feedConfigV1Content)).not.toThrow();
-    }
-  });
-
-  test('should decode v2 feeds config successfully', async () => {
-    {
-      expect(
-        async () =>
-          await decodeJSON({ name: feedConfigV2 }, NewFeedsConfigSchema),
-      ).not.toThrow();
-    }
-
-    {
-      const feedConfigV2Content = await readJSON({ name: feedConfigV2 });
-      expect(() => decodeNewFeedsConfig(feedConfigV2Content)).not.toThrow();
-    }
-  });
-
-  test('should decode v1 chainlink compatibility config successfully', async () => {
-    {
-      expect(
-        async () =>
-          await decodeJSON(
-            { name: chainlinkCompatibilityV1 },
-            ChainlinkCompatibilityConfigSchema,
-          ),
-      ).not.toThrow();
-    }
-
-    {
-      const chainlinkCompatibilityV1Content = JSON.parse(
-        JSON.stringify(await readJSON({ name: chainlinkCompatibilityV1 })),
-      );
-      expect(() =>
-        decodeChainlinkCompatibilityConfig(chainlinkCompatibilityV1Content),
-      ).not.toThrow();
-    }
-  });
-
-  test('should decode v2 chainlink compatibility config successfully', async () => {
-    {
-      expect(
-        async () =>
-          await decodeJSON(
-            { name: chainlinkCompatibilityV2 },
-            ChainlinkCompatibilityConfigSchema,
-          ),
-      ).not.toThrow();
-    }
-
-    {
-      const chainlinkCompatibilityV2Content = JSON.parse(
-        JSON.stringify(await readJSON({ name: chainlinkCompatibilityV2 })),
-      );
-      expect(() =>
-        decodeChainlinkCompatibilityConfig(chainlinkCompatibilityV2Content),
-      ).not.toThrow();
-    }
-  });
-
-  test('should decode v1 evm contracts deployment config successfully', async () => {
-    {
-      expect(
-        async () =>
-          await decodeJSON(
-            { name: evmContractsDeploymentV1 },
-            DeploymentConfigSchemaV1,
-          ),
-      ).not.toThrow();
-    }
-
-    {
-      const evmContractsDeploymentV1Content = JSON.parse(
-        JSON.stringify(await readJSON({ name: evmContractsDeploymentV1 })),
-      );
-      expect(() =>
-        decodeDeploymentConfigV1(evmContractsDeploymentV1Content),
-      ).not.toThrow();
+  test('should decode all config files successfully', async () => {
+    for (const configName of keysOf(configFiles)) {
+      expect(() => readConfig(configName)).not.toThrow();
```

Comment:
Add [onTestFailed](https://vitest.dev/api/#ontestfailed) and log the name of the file for which the decoding failed